policy/bundle.go

```diff
@@ -1106,3 +1106,18 @@ func translateGroupUIDs(ownerMrn string, policyObj *Policy, uid2mrn map[string]s
 
 	return nil
 }
+
+// Takes a query pack bundle and does the following:
+// 1. Merges all queries from the two bundles together
+// 2. Merges all props from the two bundles together
+// 3. Puts all query packs in the policy bundle
+// 4. Convert all query packs to policies with a data only scoring system. Packs are still kept
+func (p *Bundle) MergeQueryPackBundle(bundle *explorer.Bundle) {
```

Comment:
@preslavgerchev What if we changed this into a more generic `Merge(other *Bundle)` and instead have another `FromQueryPack(*explorer.Bundle)` function? It does require one more function, but they would be more specialized in their use-cases i.e. merge bundles together and convert any explorer.Bundle that we may receive. We currently have the `aggregateBundles` function in this file, which could be converted + exported into a `Merge(other *Bundle)` function. It also needs to combine the `Packs` field which is missing right now.

Reply:
Sounds good!
Then I assume we could chain this into something like `bundle.Merge(FromQueryPack(explorerBundle))` where needed. I will work on that